mumlib/src/command.rs

```diff
@@ -53,6 +53,7 @@ pub enum CommandResponse {
     ServerConnect {
         welcome_message: Option<String>,
     },
+    ServerCertReject,
```

Comment:
Maybe this should be encoded in some part of the error variant, rather than being treated as an Ok variant?